blockstore/splitstore/splitstore.go

```diff
@@ -164,7 +164,7 @@ type SplitStore struct {
 	path string
 
 	mx          sync.Mutex
-	warmupEpoch abi.ChainEpoch // protected by mx
+	warmupEpoch int64          // atomically accessed
```

Comment:
Can't tell if this int is properly aligned, probably use the `atomic.Int64` type, which is way safer/harder to mess up.